backend/fms_core/viewsets/library.py

```diff
@@ -183,12 +184,12 @@ def summary(self, _request):
         Returns summary statistics about the current set of libraries in the
         database.
         """
+        # Creates a dictionary of libraries of the form {library_id : library_type_id} which removes duplicates (pools)
+        libraries = self.queryset.values('id', 'is_pooled', 'derived_samples__library__library_type')
+        libraries_type = {l['id']: l['derived_samples__library__library_type'] for l in libraries if not l['is_pooled']}
         return Response({
-            "total_count": self.queryset.all().count(),
-            "library_type_counts": {
-                 c["derived_samples__library__library_type"]: c["derived_samples__library__library_type__count"]
-                 for c in self.queryset.values("derived_samples__library__library_type").annotate(Count("derived_samples__library__library_type"))
-            },
+            "total_count": len(libraries),
```

Comment:
The total count doesn't work as far as i can see ...  the pooled libraries with more than a single type appear multiple time.

Reply:
Libraries_type doesn't count the libraries that are pools.

Reply:
The new changes should remove the duplicates.

Reply:
We need to think about all the cases cause I think this need to be tested thoroughly

Reply:
Eheh i think building a dictionnary with all libraries will likely become very slow with time (with libraries). You are making it more complicated that it needs to be. Maybe try ".exclude(is_pooled=True)" to the library_type_count and ".filter(is_pooled=True).values("id") for the pooled libraries. You get the the total by doing the sum... or you could get the total by removing the filter. The distinct is likely already included in the endpoint queryset already.